test/e2e/scenarios/add_cluster.go

```diff
@@ -0,0 +1,62 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("[Dataplane] Adding existng network policy to newly added cluster", func() {
```

Comment:
existng -> existing :)

Reply:
ditto: at couple of places in the files.

Reply:
Done